rmgpy/data/thermo.py

```diff
@@ -1616,6 +1692,80 @@ def __addPolyRingCorrectionThermoDataFromHeuristic(self, thermoData, polyring):
                 # Typically this comment is very short and also very helpful to check if the ring correction is calculated correctly.
 
 
+    def getBicyclicCorrectionThermoDataFromHeuristic(self, bicyclic):
+
+        # saturate if the bicyclic has unsaturated bonds
+        # otherwise return None
+        bicyclic_submol = convertRingToSubMolecule(bicyclic)[0]
+        saturated_bicyclic_submol, alreadySaturated = saturateRingBonds(bicyclic_submol)
+
+        if alreadySaturated:
+            return None
+        # split bicyclic into two single ring submols
+        single_ring_submols = splitBicyclicIntoSingleRings(bicyclic_submol)
+
+        # split saturated bicyclic into two single ring submols
+        saturated_single_ring_submols = splitBicyclicIntoSingleRings(saturated_bicyclic_submol)
+
+        # apply formula: 
+        # bicyclic correction ~= saturated bicyclic correction - 
+        # saturated single ring corrections + single ring corrections
+
+        estimated_bicyclic_thermodata = ThermoData(
+            Tdata = ([300,400,500,600,800,1000,1500],"K"),
+            Cpdata = ([0.0,0.0,0.0,0.0,0.0,0.0,0.0],"J/(mol*K)"),
+            H298 = (0.0,"kJ/mol"),
+            S298 = (0.0,"J/(mol*K)")
+        )
+        
+        saturated_bicyclic_thermoData = self.__addRingCorrectionThermoDataFromTree(None, 
+                self.groups['polycyclic'], saturated_bicyclic_submol, saturated_bicyclic_submol.atoms)[0]
+        
+        estimated_bicyclic_thermodata = addThermoData(estimated_bicyclic_thermodata,
+                                                    saturated_bicyclic_thermoData , 
+                                                    groupAdditivity=True)
+
+        estimated_bicyclic_thermodata.comment = "Estimated bicyclic component: " + saturated_bicyclic_thermoData.comment
+
+        for submol in saturated_single_ring_submols:
+            
+            if not isAromaticRing(submol):
+                aromaticBonds = findAromaticBondsFromSubMolecule(submol)
+                for aromaticBond in aromaticBonds:
+                    aromaticBond.setOrderNum(1)
+                
+                submol.update()
+                single_ring_thermoData = self.__addRingCorrectionThermoDataFromTree(None, \
```

Comment:
Could you remove this backslash and the ones in the following lines?

Reply:
thanks, they are removed now